unitTests/ui/Combobox/ComboboxValuesTest.ts

```diff
@@ -142,7 +142,13 @@ export function ComboboxValuesTest() {
         combobox.element.querySelector('input').value = 'e';
         triggerRenderFromResponse(['hello']);
         const element = $$(comboboxValues.element).findAll('.coveo-checkbox-span-label')[0];
-        expect(element.innerHTML).toBe('h<span class="coveo-highlight">e</span>llo');
+        expect(element.innerHTML).toBe('<span>h</span><span class="coveo-highlight">e</span><span>llo</span>');
```

Comment:
I want to draw attention to this test and the one above.

Technically, there is a slight breaking change, where before, unhighlighted content appeared inside `Text` nodes. Following this PR, they appear inside `span` tags.

I don't think this should be breaking to CSS. However, if someone is doing a dom query to retrieve `span` tags, there will be a difference ...

Creating `Text` nodes in the `highlight` util is possible. The downside is the `elements` array becomes a mixed type `(Text | HtmlElement)[]`, which requires casting or type guarding inside the unit tests.

Let me know what you prefer,

Reply:
I'm fine with that